src/modules/0.0.23/aws_security_group/index.ts

```diff
@@ -271,7 +271,12 @@ class SecurityGroupMapper extends MapperBase<SecurityGroup> {
         const { groupId, region } = this.idFields(id);
         if (enabledRegions.includes(region)) {
           const client = (await ctx.getAwsClient(region)) as AWS;
-          const rawSecurityGroup = await this.getSecurityGroup(client.ec2client, groupId);
+          let rawSecurityGroup;
```

Comment:
this is just going to fail silently , not throwing any error?

Reply:
yes, but now that you question it, I think maybe it is not the best solution for what I want to solve. Let me handle it better